nautobot/extras/api/serializers.py

```diff
@@ -975,6 +976,41 @@ class JobInputSerializer(serializers.Serializer):
     task_queue = serializers.CharField(required=False, allow_blank=True)
 
 
+class JobMultiPartInputSerializer(serializers.Serializer):
+    _commit = serializers.BooleanField(required=False, default=None)
+    _schedule_name = serializers.CharField(max_length=255, required=False)
+    _schedule_start_time = serializers.DateTimeField(format=None, required=True)
+    _schedule_interval = ChoiceField(choices=JobExecutionType)
+    _schedule_crontab = serializers.CharField(required=False, allow_blank=True)
+    _task_queue = serializers.CharField(required=False, allow_blank=True)
+
+    def validate(self, data):
+
```

Comment:
any need for a `super()` call in here?

Reply:
No harm. I'll throw it in there. I was worried because there was no model backing it but then I remember I'm not using BaseModelSerializer

Reply:
I was worried that it would throw a validation error for the job data since they are keys that aren't in the serializer but they appear to pass.